data_structures/src/chain.rs

```diff
@@ -572,8 +572,61 @@ impl Hashable for SuperBlock {
     }
 }
 
+#[derive(Debug, Eq, PartialEq, Clone, Hash, ProtobufConvert)]
+#[protobuf_convert(pb = "witnet::SuperBlockVote")]
+pub struct SuperBlockVote {
```

Comment:
should we add the superblock index? That could be useful to compare new superblocks with the old ones

Reply:
We can try that.